src/config.ts

```diff
@@ -193,6 +193,7 @@ export interface ExtensionConfigurationSettings {
     loadCompileCommands: boolean;
     configureOnOpen: boolean | null;
     configureOnEdit: boolean;
+    deleteBuildDirOnCleanConfigure: boolean | null;
```

Comment:
```suggestion
    deleteBuildDirOnCleanConfigure: boolean;
```

We don't need to have `null` here